rqt_shell/src/rqt_shell/shell.py

```diff
@@ -85,9 +85,23 @@ def __init__(self, context):
         super(Shell, self).__init__(context)
         self._context = context
         self.setObjectName('Shell')
+        self._args = self._parse_args(context.argv())
 
         self._widget = None
 
+    def _parse_args(self, argv):
+        parser = argparse.ArgumentParser(prog='rqt_shell')
+        Shell.add_arguments(parser)
+        return parser.parse_args(argv)
+
+    @staticmethod
+    def add_arguments(parser):
+        group = parser.add_argument_group('Options for rqt_shell plugin')
+        group.add_argument('-i', '--init-script',
+                help="sources this script after startup in $SHELL.  \
+                Doesn't work for SimpleShell (or Qt5). \
```

Comment:
Please remove the `Qt5` note since it would be feasible to get another backend working on Qt 5.